pkg/background/update_request_controller.go

```diff
@@ -195,8 +189,10 @@ func (c *Controller) syncUpdateRequest(key string) error {
 		return nil
 	}
 	logger.V(3).Info("UR is marked successfully", "ur", ur.GetName(), "resourceVersion", ur.GetResourceVersion())
-	if err := c.processUR(ur); err != nil {
-		return fmt.Errorf("failed to process UR %s: %v", key, err)
+	if ur.Status.State == kyvernov1beta1.Pending {
```

Comment:
this will not allow the clone sync to be done on completed URs , unless we set the pending on update source resource ?

Reply:
is this different than what we had before ?

Reply:
Yes, it is, my bad.
Shall we process the UR whatever the state ?

Reply:
Just verified locally, we are Updating the UR with pending state in case of any updates/changes, so this works

Reply:
It looks like the check broke the tests so I removed it, will try again with the test to be sure.

Reply:
I'm now checking in the reconciler code.